docs/alerting/email-destination.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: default
+title: Destination type - Email
+nav_order: 1
+parent: monitors
```

Comment:
Parent should be Alerting. This page does not show up in the lefthand navigation.

Reply:
The parent here is Monitor, and not Alerting, that's why it doesn't show up. But I'll see how to have it show up in the left nav